zaza/openstack/charm_tests/vault/setup.py

```diff
@@ -83,7 +87,16 @@ def auto_inititialize(cacert=None):
         root_ca=cacertificate,
         allowed_domains='openstack.local')
 
-    validate_ca(cacertificate)
+    if validation_application:
+        validate_ca(cacertificate, application=validation_application)
+
+
+auto_initialize_no_validation = functools.partial(
+    auto_initialize,
+    validation_application=None)
+
+
+auto_inititialize = functools.partial(auto_initialize)
```

Comment:
I'll fix the one place that calls this (keystone-saml-mellon). So we should not need the misspelled version.